dex/src/main/resources/application.conf

```diff
@@ -694,6 +694,12 @@ waves.dex {
   }
 }
 
+# Config paths to exclude from checking for unused properties
+# This check runs at CLI command check-config and as a part of checking server at check-server command
+# Split it by ;
+# Write property path without waves.dex prefix
+waves.dex.unchecking-configs = "events-queue.kafka.consumer.client;events-queue.kafka.producer.client;events-queue.kafka.servers"
```

Comment:
Let's:
1. Rename this to `waves.dex.cli.ignore-unused-properties`
2. Use the typesafe's config lists functionality:
```
waves.dex.unchecking-configs = [
  "events-queue.kafka.consumer.client",
  "events-queue.kafka.producer.client",
  "events-queue.kafka.servers"
]
```

Reply:
Done